app/views/items/edit.html.erb

```diff
@@ -13,20 +13,22 @@
   </h4>
 
   <%= form_for @item, data: { live_guarded: true } do |f| %>
+    <% item_deleted = @item.deleted? %>
```

Comment:
I feel like variables in the view tend to be messy and clutter up the view, so I try to avoid them as much as possible. I think `item_deleted` is not significantly enough better than `@item.deleted?` to warrant adding the variable. I would suggest just using `@item.deleted?`.